lib/CMakeLists.txt

```diff
@@ -391,12 +392,11 @@ endif()
 ########################################################################
 target_sources(Pothos PRIVATE Util/Builtin/TestCompilerSupport.cpp)
 
-if(CMAKE_COMPILER_IS_GNUCXX)
-    target_sources(Pothos PRIVATE Util/Builtin/GccCompilerSupport.cpp)
-endif()
-
-if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-    target_sources(Pothos PRIVATE Util/Builtin/ClangCompilerSupport.cpp)
+if(CMAKE_COMPILER_IS_GNUCXX OR (CMAKE_CXX_COMPILER_ID MATCHES "Clang"))
```

Comment:
If the file is meant to be used for both GCC and Clang, the name should be updated accordingly.

Reply:
I couldnt think of a name. Clang copied gcc so hard that its identical command line.

CMAKE_CXX_COMPILER_ID might be useful, but it gives you "GNU" for gcc which is a little weird.

Reply:
Could be as simple as GccClangCompilerSupport.cpp